Signal/translations/TRANSLATIONS.md

```diff
@@ -37,7 +37,7 @@ the same, but in Finnish the noun/verb are distinct.
     /* Button label to archive the current conversation */
     ARCHIVE_ACTION="Archive"
 
-#### Finish
+#### Finnish
```

Comment:
I think here is no change in spell of finish.   'Finnish' & 'Finish' have different  meanings.

Reply:
True, I don’t know why I was thinking about the language. Thanks for catching that!

Reply:
Actually, here the author was giving examples about translating to Finnish the language, thus 'Finnish' with two 'n's. You can see that with the strings contained in that subsection: 'Arkisto' is Finnish for 'Archive'.